sched/task/task_create.c

```diff
@@ -77,12 +77,15 @@ int nxthread_create(FAR const char *name, uint8_t ttype, int priority,
                     FAR char * const argv[], FAR char * const envp[])
 {
   FAR struct task_tcb_s *tcb;
+  volatile size_t size;
```

Comment:
why add volatile

Reply:
It is for the "array-out-bounds" warning.

Reply:
but it isn't good to add const to local variable

Reply:
This approach is the only way I am aware of now that doesn't use `#pragma` and can turn off the `array out-of-bounds` warning.

do you have a better way to turn off the `array-out-bounds` warning?